File: patterns/pattern-8/pattern8.js
process.stdin.on('data',(userInput)=>{
    n = parseInt(userInput)
    for(let i = 1;i<=n;i++){
        total = n * 2 - i
        space = i -1
        star = total - space
        for(let j = 1;j<=total;j++){
            if(j<=space){
        process.stdout.write("  ");
      
            }else{
        process.stdout.write("* ");
            }
        }
        process.stdout.write("\n");
    }
    process.exit();
    })

File: patterns/pattern-8/pattern8.py
n = int(input())
for i in range(1,n+1):
    total = n * 2 - i
    space = i - 1
    star = total - space
    for j in range(1,total + 1):
        if j<=space:
            print(" ",end=" ")
        else:
            print("*",end=" ")
    print()